packages/zosfiles/src/api/methods/create/doc/ICreateDataSetOptions.ts

```diff
@@ -123,4 +123,9 @@ export interface ICreateDataSetOptions {
      * @type {string}
      */
     size?: string;
+    /**
+     * The name of another data set from which to copy attributes
+     * @type {string}
+     */
+    like?: string;
```

Comment:
When "like" is used on `ICreateDataSetOptions`, no other properties are needed. Should we make every property on the interface optional? Currently "primary" and "lrecl" are required.

This may be a question for @zFernand0 🙂 If we change required values to be optional, we may need to add validation logic where those values were being used before.

Reply:
Sounds to me that we may need a separate `object` i.e. `zowe files create data-set-like` since we usually aim to avoid confusion with what options are conditionally required and/or mutually exclusive.

I do agree that if we decide to make them all optional, additional validation (at the API level) will be required in order to guarantee the same behavior.

Reply:
I like the first idea :) But @tjohnsonBCM I will wait for your input before implementing anything...

Reply:
@katelynienaber I agree with the suggestion to define a new interface (perhaps `ICreateDataSetLikeOptions`?), to minimize confusion about which options on the interface are required.

If the "allocate like" functionality is implemented as a CLI command in the future, it would make sense for it to be a new command (`zowe files create data-set-like` as Fernando mentioned). This is consistent with creating a new interface.

Reply:
@tjohnsonBCM @zFernand0 I added a new interface and a separate function in the Create class, please check if this meets your specifications ^^

Reply:
Should "like" be removed from `ICreateDataSetOptions` now?

Reply:
Removed!